scripts/auto_resolve_halt.sh

```
@@ -0,0 +1,763 @@
+        const lastBlock = db.blocks.find().sort({_id:-1}).limit(1).next();
+        if (!lastBlock) {
+            print('ERROR: No blocks found');
+            quit(1);
+        }
+        const oldHeight = lastBlock._id;
+        const newHeight = oldHeight - $remove_count;
+        if (newHeight < 0) {
+            print('ERROR: Cannot remove $remove_count blocks, only have ' + (oldHeight+1));
+            quit(1);
+        }
+        const isDryRun = "'"$DRY_RUN"'" === "1";
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)